src/screens/address/address.vue

```diff
@@ -18,10 +19,12 @@
       </field>
     </header>
 
-    <h2>Transactions</h2>
-    <div class="transactions">
-      <transaction :key="t.hash" v-for="t in account.transactions" :transaction="t"/>
-    </div>
+    <!--<h2>Transactions</h2>-->
+    <!--<div class="transactions">-->
+      <!--<transaction :key="t.hash" v-for="t in account.transactions" :transaction="t"/>-->
+    <!--</div>-->
+    <!--TODO implement transactions after middleware end points are available-->
```

Comment:
As far as we are using VCS you can completely remove this code and add an issue pointing to this commit if this functionality is going to reimplemented.